### ISSUE TITLE
Compile (syntax only) creates an error if the preview version of Visual Studio is also installed

### ISSUE THREAD

**Issue by lohrun** [Caphyon/clang-power-tools#314]
The clang Compile (syntax only) produces the following error on my computer. From what I am gathering of the error message, the  clang-build.ps1 script has an issue if more than one Visual Studio version is found on the computer which is the case when the current and preview version are installed side by side.

I have installed Visual Studio 15.6.4 and Visual Studio 15.7.0 Preview. I am using Clang Power Tool from the current Visual Studio 15.6.4.

```
PROJECT: xxx_UnitTest.vcxproj
Get-Item : Cannot find path 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise C:\Program Files 
(x86)\Microsoft Visual Studio\Preview\Enterprise\VC\Tools\MSVC\' because it does not exist.
At 
...\APPDATA\LOCAL\MICROSOFT\VISUALSTUDIO\15.0_ABE6A81E\EXTENSIONS\cbmba442.jdn\clang-build.ps1:601 
char:12
+   return ((Get-Item "$(Get-VisualStudio-Path)\VC\Tools\MSVC\" | Get-C ...
+            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Program File...\VC\Tools\MSVC\:String) [Get-Item], ItemNotFoundExcep 
   tion
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetItemCommand

```

**Comment by cpp-red-lion** [comment]
Hi,

Sorry for the late reply.

Thanks for bringing this issue to our attention.   
We'll look into it and update this thread.

**Comment by hero101111** [comment]
Hi again,

This issue should be fixed as of ClangPowerTools v3.6
Thanks for the feedback.

Regards,
Gabriel

**Comment by lohrun** [comment]
Indeed, I don't have any problem anymore. Thank you very much !
